src/packages/ufm/components/manifests.ts

```diff
@@ -15,6 +15,9 @@ export const manifests: Array<ManifestUfmComponent> = [
 		api: () => import('./localize/localize.component.js'),
 		meta: { alias: 'umbLocalize', marker: '#' },
 	},
+	/*
+	 * @deprecated since version 15.0-rc3
```

Comment:
This isn't correct, as the UFM component (manifest) `Umb.Markdown.ContentName` isn't being deprecated, only the `~` marker prefix will be deprecated. We'll still support `{umbContentName: alias}`.

Reply:
oh yes, I though it as one registrating pr. 'marker' but I see it now. I have reverted.